murageden/bootcamp
# Create a Flask RESTful API

#### What does this PR do?
Adds a FLask RESTful API that allows Creation, Reading, Updating and Deletion of client data without persitence of this data

#### Description of Task to be completed?
Have the following API endpoints working:
`POST /api/v1/auth/register`
`POST /api/v1/auth/login`
`POST /api/v1/auth/reset-password`
`POST /api/v1/businesses`
`POST /api/v1/businesses/<businessId>`
`DELETE /api/v1/businesses/<businessId>`
`GET /api/v1/businesses`
`GET /api/v1/businesses/<businessId>`
`POST /api/v1/businesses/<businessId>/reviews`
`GET /api/v1/businesses/<businessId>/reviews`

#### How should this be manually tested?
Clone the repo, run the application on localhost then test the routes on postman.
Check README for details

#### Any background context you want to provide?
**NEW** Changing user password requires the client to provide both a new password and the current password as well

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2153383
All finished stories
